src/math/lp/bound_analyzer_on_row.h

```diff
@@ -361,7 +329,20 @@ public :
         }
     }
 
-
+    void analyze_eq() {
+        switch (m_bp.lp().settings().cheap_eqs()) {
+        case 0:
+            return;
+        case 1:
+            m_bp.cheap_eq_tree(m_row_index);
+            break;
+        case 2:
+            m_bp.cheap_eq_table(m_row_index);
+            break;
+        default:
+            UNREACHABLE();
```

Comment:
to avoid some fuzzer filing a bug by miss-configuring z3, just return here (or raise an exception)